Rephrase API settings (#264)
meilisearch/documentation: references/ranking_rules.md

_Child route of the [settings route](/references/settings.md)._

> Ranking rules are predefined rules applied to the search results in order to improve their relevance. They are customizable so the results meet your user's needs as close as possible. Ranking rules are applied in a default order which can be changed in the settings.

Reviewer: predefined or built-in?
```suggestion
Ranking rules are built-in rules that **ensure relevancy in search results**. Ranking rules are applied in a default order which can be changed in the settings. You can add or remove rules and change their order of importance.
```

Author: I will use built-in everywhere

#### Body

An array that contain ranking rules sorted by order of importance.

> To add your own ranking rule, you have to communicate either `asc` for ascending order or `dsc` for descending order followed by the field name in brackets.

Reviewer: ```suggestion
To add your own ranking rule, you have to communicate either `asc` for ascending order or `desc` for descending order followed by the field name in brackets.
```

- To apply an **ascending sorting** (results sorted by increasing value of the attribute): `asc(attribute_name)`

> - To apply a **descending sorting** (results sorted by decreasing value of the attribute): `dsc(attribute_name)`

Reviewer: ```suggestion
- To apply a **descending sorting** (results sorted by decreasing value of the attribute): `desc(attribute_name)`
```

### Examples

```bash
$ curl \
  -X POST 'http://localhost:7700/indexes/movies/settings/ranking-rules' \
  --data '[
      "typo",
      "words",
      "proximity",
      "attribute",
      "wordsPosition",
      "exactness",
      "asc(release_date)",

>       "dsc(rank)"

Reviewer: ```suggestion
      "desc(rank)"
```

#### Default value

> An array that contain [built-in ranking rules](/guides/main_concepts/relevancy.md#order-of-the-rules) sorted by order of importance.

Reviewer: Not sure
```suggestion
An array that contains [built-in ranking rules](/guides/main_concepts/relevancy.md#order-of-the-rules) sorted by order of importance.
```
